e2e/src/reasonCodesController/reasonCodes.spec.ts

```diff
@@ -0,0 +1,227 @@
+import { ReasonCodesController } from 'advanced-billing-sdk';
+import { createClient, createInvalidClient } from '../config';
+import { deleteAllReasonCodes } from './utils';
+
+describe('Reason codes', () => {
```

Comment:
non blocking: the structure of the test looks like you partially could take advantage of [test.each](https://jestjs.io/docs/api#testeachtablename-fn-timeout)

Reply:
that util is new for me, sounds like an abstraction of common scenarios, I will take a look